coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -2,28 +2,32 @@
 
 
 @generate_repr()
-@generate_eq("documentation", "language", "docstyle", "marker", "range")
+@generate_eq("documentation", "language", "docstyle",
+             "indent", "marker", "range")
 class DocumentationComment:
```

Comment:
commit message, you have still mistyped `indentatiion` :)
(as you need to change this commit anyway :P)